### DISCUSSION TITLE
Consulta - Parcial Muertos Andantes

### DISCUSSION THREAD

**Discussion by tomaslcarballo** [dds-jv/foro#329] [Consultas de persistencia]
Hola, estuve haciendo este parcial y ví que en la resolución ponen dentro de la clase caminantes el lugar y sobreviviente una @ManyToOne. En mi caso cuando lo resolví hice lo contrario, en Lugar agregue el atributo caminantes con @OneToMany y en la clase predador(es lo misma que sobreviviente en la solución) un atributo caminantes con la misma anotación.

Mi duda es cual es la diferencia entre estas 2 formas de encarar esta parte? Hay una que es más recomendable que la otra?

![image](https://user-images.githubusercontent.com/62314468/142515059-681d0373-d2e9-47f7-ac31-1ed0f7c52dfc.png)
![image](https://user-images.githubusercontent.com/62314468/142515088-727186b7-f0a8-4dfb-b478-6bd0722b67d2.png)
![image](https://user-images.githubusercontent.com/62314468/142515111-0d1f09d1-1aaa-4c68-8189-82b22666ba08.png)

Muchas gracias.

**Comment by Franpanozzo** [comment]
Buenas tomas, yo también recién resolví el parcial recién y estoy casi seguro que los atributos Lugar y Sobreviviente no van en caminante, ya que como justamente indica el diagrama, el Lugar y el Predador son los que tienen una lista de caminantes. 
Por lo tanto lo resolví como vos con la diferencia de que en la relacion en el atributo caminantes del Predador la hice ManyToMany pensando que porque no varios predadores pueden conocer a los mismos caminantes

**Reply by tomaslcarballo** [reply]
Genial, muchas gracias Fran por la ayuda!
